scanner/config.go

```diff
@@ -12,6 +12,36 @@ import (
 	"github.com/bitrise-io/go-utils/sliceutil"
 )
 
+const otherProjectType = "other"
+
+type scanResultStatus string
+
+const (
+	// in case DetectPlatform() returned error, or false
+	scanResultNotDetected scanResultStatus = "scanResultNotDetected"
+	// in case DetectPlatform() returned true, but Options() or Config() returned an error
+	scanResultDetectedWithErrors scanResultStatus = "scanResultDetectedWithErrors"
+	// in case DetectPlatform() returned true, Options() and Config() returned no error
+	scanResultDetected scanResultStatus = "scanResultDetected"
+)
+
+type scannerOutput struct {
+	scanResult scanResultStatus
```

Comment:
i would call this variable simply `status`

Reply:
fixed